app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/vaultunlockednavbar/VaultUnlockedNavBarViewModel.kt

```diff
@@ -77,6 +92,14 @@ class VaultUnlockedNavBarViewModel @Inject constructor(
     // #endregion BottomTabViewModel Action Handlers
 }
 
+/**
+ * Models state for the [VaultUnlockedNavBarViewModel].
+ */
+data class VaultUnlockedNavBarState(
+    @StringRes val vaultNavBarLabelRes: Int,
+    @StringRes val vaultNavBarContentDescriptionRes: Int,
```

Comment:
Can we use `Text` here?

Reply:
Using `Text` here is going to cause issues interacting with the `VaultUnlockedNavBarTab` which was setup with a string resource. I do agree that would be ideal but it would incur a lot more changes in the screen.